html/themes/custom/common_design_subtheme/components/rw-article/rw-article.css

```diff
@@ -21,15 +25,19 @@
   position: relative;
   display: flex;
   flex-direction: column;
-  padding: 24px 0;
-  border-top: 1px solid #e6ecef;
+  padding: 0 0 24px 0;
+  border-bottom: 1px solid #e6ecef;
+}
+.rw-river-article:last-child {
```

Comment:
Removing the bottom padding affects sections like the maps and infographics on the country/disaster pages:

<img width="941" alt="maps-cards" src="https://user-images.githubusercontent.com/696348/129661202-8ff6cce9-37f3-4ca9-a5e0-81045baeb179.png">

Reply:
fixed